examples/tracing/tcpv4listen.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
```

Comment:
I think these should be /usr/bin/python instead of /usr/bin/env, it will reduce the PATH searching and speed up startup time. Or at least that is what @brendangregg argued when he critiqued my use of env in an old patch :)